docs/user-guide/debugging-services.md

```diff
@@ -555,6 +555,63 @@ There are three things to check:
   Engine doesn't do port remapping, so if your application serves on 8080,
   the `containerPort` field needs to be 8080.
 
+### A Pod cannot reach itself via Service IP
+
+This mostly happens when `kube-proxy` is running in `iptables` mode and `Pod`s
+are connected with bridge network. There is `--hairpin-mode` flag in `Kubelet`.
```

Comment:
please reword: the kubelet exposes a `hairpin-mode` [flag] link to --hairpin-mode? that allows endpoints of a Service to loadbalance back to themselves if they should try to access their own Service VIP. The flag must either be set to `hairpin-veth`, or `promiscuous-bridge`.